domain/schema/model/sql/0018-machine.sql

```diff
@@ -57,6 +57,36 @@ CREATE TABLE machine_agent_version (
     REFERENCES architecture (id)
 );
 
+-- v_machine_agent_version provides a convenience view on the
+-- machine_agent_version reporting the architecture name as well as the id.
+CREATE VIEW v_machine_agent_version AS
+SELECT
+    m.name,
+    mav.machine_uuid,
+    mav.architecture_id,
+    mav.version,
+    a.name AS architecture_name
+FROM machine_agent_version AS mav
+JOIN machine AS m ON mav.machine_uuid = m.uuid
+JOIN architecture AS a ON mav.architecture_id = a.id;
```

Comment:
We only use this view once. Let's just merge it with `GetMachineRunningAgentBinaryVersion`'s query.

Reply:
I have updated with a comment explaining the why. Essentially working around some SQLAir stuff for this.